pkg/client/src/app/pages/applications/analysis-wizard/set-options.tsx

```diff
@@ -165,22 +170,42 @@ export const SetOptions: React.FunctionComponent = () => {
             ))}
           </Select>
         </FormGroup>
-        <FormGroup label="Excluded rules tags" fieldId="excluded-rules">
+        <FormGroup
+          label="Excluded rules tags"
+          fieldId="ruleTagToExclude"
+          validated={getValidatedFromError(errors.ruleTagToExclude)}
+          helperTextInvalid={errors?.ruleTagToExclude?.message as string}
```

Comment:
Why do we need this assertion here

Reply:
This is needed to display the error message as part of the FormGroup.

Reply:
I meant the type assertion.

Reply:
Oh I see.
Because the message is type any so it makes sense to cast it as string.
That said the target is a ReactNode which accepts any. So I can remove it